src/gf/seismosizer.py

```diff
@@ -1692,20 +1740,38 @@ class RectangularSource(SourceWithDerivedMagnitude):
         optional=True,
         default=1)
 
```

Comment:
As mentioned above maybe already finalize the attributes and allow the centre of the gaussian to be optionally shifting as well...The rest of the code allows it already. Now you simply hardcode it to zero.